process.c

```diff
@@ -857,10 +857,16 @@ pst_inspect(VALUE st)
 
 /*
  *  call-seq:
- *     stat == other   -> true or false
+ *    stat == other -> true or false
+ *
+ *  Returns whether the value of #to_i == +other+:
+ *
+ *    Process.spawn('cat /nop') # => 1156807
+ *    Process::Status.wait      # => #<Process::Status: pid 1156807 exit 1>
```

Comment:
We can simplify these examples by using backticks to run it synchronously rather than using `Process.spawn` then `Process.wait`.

Reply:
Done.